tools/cli/go-whisk-cli/commands/property.go

```diff
@@ -87,6 +90,14 @@ var propertySetCmd = &cobra.Command{
                     map[string]interface{}{"ok": color.GreenString("ok:"), "auth": boldString(auth)}))
         }
 
+        if insecure := flags.property.insecureSet; len(insecure) > 0 {
+            client.Config.Insecure = strings.ToLower(insecure) == "yes" || strings.ToLower(insecure) == "true"
```

Comment:
Do we always want to set the insecure property to false if values of `yes` or `true` are not passed?